src/trix/models/composition.coffee

```diff
@@ -217,11 +232,14 @@ class Trix.Composition extends Trix.BasicObject
       @removeCurrentAttribute(attributeName)
 
   canSetCurrentAttribute: (attributeName) ->
+    isTerminalBlock = @getBlock()?.isTerminalBlock()
```

Comment:
Looks like `isTerminalBlock` is only used once whereas `@getBlock()?` is called several times. I'd set `block = @getBlock()` here instead and reuse that. And, is there a case where no block is present or can we drop the existential operator?